pyEpiabm/pyEpiabm/tests/test_core/test_cell.py

```diff
@@ -39,6 +48,12 @@ def test_number_infectious(self):
         person.update_status(InfectionStatus.InfectMild)
         self.assertEqual(cell.number_infectious(), 1)
 
+    def test_set_loc(self):
+        cell = pe.Cell()
+        self.assertEqual(cell.location, (0, 0))
+        cell.set_location((3.0, 3.0))
+        self.assertEqual(cell.location, (3.0, 3.0))
```

Comment:
```suggestion
        cell.set_location((3.0, 2.0))
        self.assertEqual(cell.location, (3.0, 2.0))
```

This might be my code, but correcting my own mistakes it would definitely be safer to have two different values in this test (just in case the indices have been mixed up in the function that sets the location)